### PR TITLE
miguel changes

### PR DESCRIPTION
**Pull request by mchacong**

### CODE REVIEW

File: modules/amp-360/pages/amp-360-plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+:current-module: amp-360
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpVR360Sample.zip
+:plugin-common-name:360
+
+= Amp 360
+
+The Amp 360 plugin allows you to play corretly content formatted for virtual reality purposes.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-360/pages/amp-360-plugin.adoc]: @mchacong  Miguel,

typo:
correctly

File: modules/amp-360/pages/amp-360-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+You can check out our basic implementation, please refer to our application samples:
+
+* {amp-plugin-sample-ios}[AMP {plugin-common-name} application sample for iOS].
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-360/pages/amp-360-plugin.adoc:13]: @mchacong  
Miguel,
what is the link, or indication for the user to find/download corresponding samples.?

File: modules/amp-360/pages/amp-360-plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+To install the Amp360 just manually add the framework or add it through CocoaPods with the same pod name `Amp360`. The steps are the same as with the installation of {amp-player-docs}[our documentation].
+
+== Implementation
+
+You need a working implementation of the AmpPlayer before using this plugin. If you need the implementation guide for AmpPlayer please refer to {amp-player-docs}[our documentation].
+
+=== Import
+
+The first step is to use the 360 plugin is to import the plugin.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-360/pages/amp-360-plugin.adoc]: @mchacong 
Miguel, 
syntax issue
the first step for using the Amp-360 plugin is ...

File: modules/amp-downloader/pages/amp-downloader-plugin.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+To install the AmpDownloader just manually add the framework or add it through CocoaPods with the same pod name `AmpDownloader`. The steps are the same as with the installation of {amp-player-docs}[our documentation].
+
+== Implementation
+
+You need a working implementation of the AmpPlayer before using this plugin. If you need the implementation guide for AmpPlayer please refer to {amp-player-docs}[our documentation].
+
+=== Import
+
+The first step is to use the Downloader is to import the plugin.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-downloader/pages/amp-downloader-plugin.adoc]: same syntax issue , here and others.. @mchacong 
Miguel

File: modules/amp-downloader/pages/amp-downloader-plugin.adoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    return documentDirectory.appending("/AmpDownloads.plist")
+}()
+
+func getDownloads() -> [AmpDownload] {
+    var array:[AmpDownload] = []
+    if let data = FileManager.default.contents(atPath: path),
+        let decodedArray = try? PropertyListDecoder().decode(Array<AmpDownload>.self, from: data) {
+        array = decodedArray
+    } else {
+        array = []
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-downloader/pages/amp-downloader-plugin.adoc]: maybe this else is not required as array is already empty. 
just a note.

File: modules/amp-downloader/pages/amp-downloader-plugin.adoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+}
+----
+
+=== Other functionalities
+
+There are also other methods for managing your downloads:
+
+`AmpDownloadManager.pauseDownload`: Pauses a download
+`AmpDownloadManager.resumeDownload`: Resumes a download
+`AmpDownloadManager.cancelDownload`: Deletes the download
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-downloader/pages/amp-downloader-plugin.adoc]: is the cancel (stop while downloading the asset) the same method used for delete (remove already downloaded asset). ??

File: modules/amp-firebase/pages/amp-firebase-plugin.adoc [hunk 7]
```diff
@@ -0,0 +1,190 @@
+:current-module: amp-mux
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-docs: https://firebase.google.com/docs
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpFirebaseSample.zip
+:plugin-common-name:Firebase
+
+= Firebase Analytics
+
+The AmpFirebase plugin offers a quick and escalable way to integrate AmpPlayer with {amp-plugin-docs}[Google Firebase]. This plugin enables the reporting of data collected by AMP in your application to your own instance of Firebase, efectively tracking your video activity. It will provide you the liberty of tracking the events you are interested in, keeping your logs with only the ipmortant information.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-firebase/pages/amp-firebase-plugin.adoc]: typo ipmortant

File: modules/amp-firebase/pages/amp-firebase-plugin.adoc [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+[source,swift]
+----
+class firebaseArgs: AmpFirebaseArgs {
+    override func getGlobalArgs() -> [String : Any] {
+        return [ CUSTOM_KEY : VOD_STREAM_NAME ]
+    }
+}
+----
+
+Yo can override as many methods as you want and even add your own logic on how the information changes across time. There are ways to add additional arguments for every event, for a single category or even for a specific event.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-firebase/pages/amp-firebase-plugin.adoc]: typo
Yo

File: modules/amp-meta/pages/amp-facebook-plugin.adoc [hunk 9]
```diff
@@ -0,0 +1,59 @@
+:current-module: amp-facebook
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-docs: https://developers.facebook.com/docs/audience-network/setting-up/platform-setup/ios/get-started/
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpFBAudienceNetworkSample.zip
+:plugin-common-name:Meta Audience Network
+
+= Meta Audience Network
+
+The AmpFBAudienceNetwork plugin offers a quick way to integrate AmpPlayer with {amp-plugin-docs}[Mux SDK]. This plugin enables the display of ads in your playback.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-meta/pages/amp-facebook-plugin.adoc]: Facebook or Mux?

File: modules/amp-yospace/pages/amp-yospace-plugin.adoc [hunk 10]
```diff
@@ -0,0 +1,63 @@
+:current-module: amp-yospace
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-docs: https://developer.yospace.com/sdk-documentation/apple/userguide/latest/en/index-en.html
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpYoSpaceSample.zip
+:amp-plugin-sample-tvos: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpYoSpaceSampleTV.zip
+:plugin-common-name:YoSpace
+
+= YoSpace Analytics
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-yospace/pages/amp-yospace-plugin.adoc]: I am confused @mchacong @orlando-amp Miguel, I thought our yospace integration is just for SSAI Ads. 

I am not aware of Yospace Analytics, 
is this file correct?

**Review by JeanCarlosChavarriaHughes — CHANGES_REQUESTED** [review]
Miguel please review the comments,

**Comment by mchacong** [comment]
push miguel changes

**Review by JeanCarlosChavarriaHughes — APPROVED** [review]

**Pull request merged**